internal/users/cache/testdata/TestDeleteUser/golden/delete_existing_user

```diff
@@ -0,0 +1,7 @@
+GroupByID: {}
```

Comment:
maybe 2 tests, one with only this one, one with multiple users, delete one of them.

Reply:
I don’t see the second tests still?

Reply:
Oh, you meant on these tests... I've added the test case to the UpdateUser tests. I'll add it here too.